### DISCUSSION TITLE
How to get rid of this error? "clang: In included file: reference to unresolved using declaration"

### DISCUSSION THREAD

**Discussion by j0ker70** [clangd/clangd#1741] [Q&A]
I have clangd attached with my neovim config. I was searching through the chromium source code. But in every file I get this error for 1 or 2 header files:
clang: In included file: reference to unresolved using declaration

![image](https://github.com/clangd/clangd/assets/28401403/74fe379c-3c17-4c37-8222-e96b4100cdb0)

How can I configure clangd to get rid of this error?

**Comment by HighCommander4** [comment]
Have you generated a `compile_commands.json` file for the project?

**Reply by j0ker70** [reply]
Yes I did. Using this command:
```
tools/clang/scripts/generate_compdb.py -p out/Default > compile_commands.json
```

**Reply by HighCommander4** [reply]
Ok, assuming that's run in the project's root directory (more specifically, that the directory that contains the compile_commands.json is an ancestor of the file being opened, chrome/browser/importer/importer_list.cc), clangd should be finding and using the compile_commands.json. (To be sure, you can check the [clangd logs](https://clangd.llvm.org/troubleshooting#gathering-logs), which includes the compile command used when a file is opened.)

The next thing to try would be to open the generated compile_commands.json, find the entry for importer_list.cc, and try running the command in that entry on the command line. Does it succeed or do you get the same error produced by clangd?
